### PR TITLE
Prevent concurrent modification when iterating request commands

### PR DESCRIPTION
**Pull request by lahma**
### Prerequisites

- [x]  I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp.Web/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [ ] I have provided test coverage for my change (where applicable)

### Description

There was an unfortunate problem in optimization PR which surfaced in OrchardCore. Commands were mutated with active iterator and that broke the contract. This fixes the problem and hopefully you could do yet another release to NuGet 🙏

### CODE REVIEW

File: src/ImageSharp.Web/Middleware/ImageSharpMiddleware.cs [hunk 1]
```diff
@@ -157,7 +157,8 @@ public async Task Invoke(HttpContext context)
             IDictionary<string, string> commands = this.requestParser.ParseRequestCommands(context);
             if (commands.Count > 0)
             {
-                foreach (string command in commands.Keys)
+                List<string> commandKeys = new List<string>(commands.Keys);
```
Comment (JimBobSquarePants) [inline src/ImageSharp.Web/Middleware/ImageSharpMiddleware.cs]: I would have used `ToArray()` here but a list will do for now.

**Comment by JimBobSquarePants** [comment]
@lahma This is certainly critical enough to warrant a new release so here you go! 😄 

https://www.nuget.org/packages/SixLabors.ImageSharp.Web/1.0.0-beta0009

**Comment by lahma** [comment]
Thanks for the prompt action and release (and sorry for the extra chore)! 👍

File: src/ImageSharp.Web/Middleware/ImageSharpMiddleware.cs (hunk 1 above)
Comment (lahma) [reply src/ImageSharp.Web/Middleware/ImageSharpMiddleware.cs]: I dread to use System.Linq in active/hot path 😉

**Pull request merged**